index.js

```diff
@@ -106,3 +106,11 @@ btnBack.addEventListener('click', function () {
         console.log(nr2);
     }
 });
+
+const showHistory = document.querySelector('#show-history');
```

Comment:
Let's rename to `btnHistory` so it's using the same pattern as the other buttons.